tests/z9999_image_bootstrap_test.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+"""Bootstrap CDT Version Test
+
+This script tests that the bootstrap command can create an image with a given version.
+
+This test contains resource intense tests that should NOT be run on every commit.
+To disable during ci testing, add `-k "not non_ci"` to your pytest command (e.g. `pytest -k "not non_ci" tests`)
```

Comment:
```suggestion
To disable during ci testing, add `-k "not expensive"` to your pytest command (e.g. `pytest -k "not expensive" tests`)
```

Reply:
This is overcome by the latest change, I believe.